packages/breakpoint-split-view/src/BreakpointSplitView.ts

```diff
@@ -10,67 +10,104 @@ export default ({ jbrequire }: { jbrequire: Function }) => {
   const { stateModel, configSchema } = jbrequire(require('./model'))
 
   class BreakpointSplitViewType extends ViewType {
-    snapshotFromBreakendFeature(
+    async snapshotFromBreakendFeature(
       feature: Feature,
-      startRegion: IRegion,
-      endRegion: IRegion,
+      view: {
+        displayRegionsFromAssemblyName?: string
+        displayedRegions: [{ start: number; end: number; refName: string }]
+      },
     ) {
       const breakendSpecification = (feature.get('ALT') || [])[0]
       const startPos = feature.get('start')
       let endPos
       const bpPerPx = 10
-      const topRegions = [{ ...startRegion }, { ...startRegion }]
-      const bottomRegions = [{ ...endRegion }, { ...endRegion }]
+      const asm = view.displayRegionsFromAssemblyName
+
+      const getCanonicalRefName = (ref: string) => {
+        if (asm) {
+          const session = getRoot(view).jbrowse
```

Comment:
getSession(view) does not produce the right thing...if I use

const { getSession } = jbrequire('@gmod/jbrowse-core/util')
const session = getSession(view)
session.getCanonicalRefName(...)

Then it gets getCanonicalRefName is not a function

Reply:
Ok, yeah, that makes sense. getCanonicalRefName isn't on the session, it's on the jbrowseModel.

Long-term, I'd like anything a package/plugin needs to be on the session, so that the session can be the single source we have to worry about keeping consistent for plugins/products. But that's part of a larger session API discussion for later. I'm fine with this for now.